README.md

```diff
@@ -1 +1,10 @@
-# xswap-analysis
\ No newline at end of file
+# Analysis and figure generation for XSwap project
+
+This repository hosts notebooks, images, and data for the XSwap project.
+The manuscript corresponding to this analysis is:
+
++ [**The probability of edge existence due to node degree: a baseline for network-based predictions**](https://github.com/greenelab/xswap-manuscript/)<br>
+  Michael Zietz, Daniel S. Himmelstein, Kyle Kloster, Christopher Williams, Michael W. Nagle, Casey S. Greene
```

Comment:
It would probably be easier to write something like "The results of these analyses are described in the manuscript titled TITLE" without an author list as that can change. Once the study is journal published we can update the citation here.